pom.xml

```diff
@@ -39,8 +39,8 @@
     <ramlfiles_util_path>${ramlfiles_path}/raml-util</ramlfiles_util_path>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <aspectj.version>1.9.6</aspectj.version>
-    <raml-module-builder.version>34.0.0</raml-module-builder.version>
-    <vertx.version>4.1.4</vertx.version>
+    <raml-module-builder.version>35.0.0</raml-module-builder.version> <!-- also update vertx.version -->
```

Comment:
RMB 35.0.1 has been released, we might as well bump the version here prior to merge